junebug/api.py

```diff
@@ -215,10 +215,17 @@ def send_message(self, request, body, channel_id):
             raise ApiUsageError(
                 'Only one of "from" and "reply_to" may be specified')
 
-        try:
-            self.service.getServiceNamed(channel_id)
-        except KeyError:
-            raise ChannelNotFound()
+        channel = yield Channel.from_id(
+            self.redis, self.config, channel_id, self.service)
+
+        if (channel.character_limit is not None
+                and len(body['content']) > channel.character_limit):
+            raise ApiUsageError(
+                'Message content %r is of length %d, which is greater than the'
+                ' character limit of %d' % (
+                    body['content'], len(body['content']),
+                    channel.character_limit)
+                )
```

Comment:
The problem with using `Channel.send_*` is that they are class methods, so we would have to send through a redis manager in the `send_*` methods.

The reason they were class methods in the first place is so that we don't need to get the channel properties from redis, but since we need to do it now anyway, maybe we should look at making them normal methods?